src/EventNew.js

```diff
@@ -0,0 +1,149 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Paper from "@mui/material/Paper";
+import Stepper from "@mui/material/Stepper";
+import Step from "@mui/material/Step";
+import StepLabel from "@mui/material/StepLabel";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+import ImageUploader from "./ImageUploader";
+import Copyright from "./Copyright";
+import LocationForm from "./LocationForm";
+import Review from "./Review";
+import Header from "./Header";
+import { addEvent } from "./Events";
+
+const steps = ["Event Details", "Event Photos", "Review Event"];
+
+const theme = createTheme({
+  palette: {
+    mode: "dark",
+  },
+});
+
+function getStepContent(step, func, event) {
+  switch (step) {
+    case 0:
+      return <LocationForm handleChange={func} event={event} />;
+    case 1:
+      return <ImageUploader event={event} />;
+    case 2:
+      return <Review event={event} />;
+    default:
+      throw new Error("Unknown step");
+  }
+}
+
+export default function Checkout() {
+  const [activeStep, setActiveStep] = React.useState(0);
+
+  const [event, setEvent] = React.useState({
+    name: "",
+    description: "",
+    location: "",
+    category: "",
+    date: "",
+    // hardcode these for now
```

Comment:
What is this supposed to represent, event template?

Reply:
These are all the fields that our DB has for an event @anthony-yeo. Our mongoDB specifies that all fields are required, so I have to hardcode some values.

Reply:
@anthony-yeo If you are actually asking about why they are all set to empty, it is because these values are set via `useState()` in react, so they are actually updated when users enter values into the form.